### PR TITLE
Improve some description in the community guide page

### PR DESCRIPTION
**Pull request by zhoujinsong**
According to the discussion in the dev mailing list: https://lists.apache.org/thread/mcr201tdw3075873y5pvr62cbzd31b5x, I would like to improve some description in the community guide page.

### CODE REVIEW

File: amoro-site/content/community/join-community.md [hunk 1]
```diff
@@ -44,32 +43,19 @@ Everyone who contributes can become an Amoro contributor. The members will provi
...
-- Consistently active in the community and actively participate in community events such as online/offline meetups and community discussions
-### Responsibilities and privileges
-- Join the community meeting and discussion
-- Mentor and guide new contributors
-- Be listed as an Amoro Active Contributor and featured on the Amoro official website
-- Be awarded an Amoro Active Contributor e-certificate
 
 ## Committer
-Committers are promoted from Active Contributors. They have the authority to merge master branches and are responsible for the planning and maintenance of Amoro. They are also active members who share their knowledge with the community.
+Committers are promoted from Contributors. They have the authority to merge master branches and are responsible for the planning and maintenance of Amoro. They are also active members who share their knowledge with the community.
```
Comment (justinmclean) [inline amoro-site/content/community/join-community.md:48]: I would suggest saying "They have the authority to commit to the project's repositories and..."

File: amoro-site/content/community/join-community.md [hunk 2]
```diff
@@ -44,32 +43,19 @@ Everyone who contributes can become an Amoro contributor. The members will provi
...
 ### How to become a Committer?
 - Have a deep understanding of Amoro's principles and future plans
 - Have the ability to deal with various issues that arise in the project promptly
 - Lead a major development, write and revise related documents
 - Be voted in by the Amoro PPMC
 ### Responsibilities and privileges
 - Mentor and guide other memberships in the community
```
Comment (justinmclean) [inline amoro-site/content/community/join-community.md:55]: this should say "other members" not "other memberships"

File: amoro-site/content/community/join-community.md [hunk 3]
```diff
@@ -84,5 +70,4 @@ PPMC members are responsible for the planning and maintenance of Amoro. They are
 - Make and approve technical design decisions
 - Define milestones and releases
 - Vote and promote new committers and PPMC members
-- Be listed as an Amoro PPMC member and featured on the Amoro official website
-- Be awarded an Amoro PPMC e-certificate
+- Be listed as an Amoro PPMC member and featured on the Amoro official website
```
Comment (justinmclean) [inline amoro-site/content/community/join-community.md:73]: Other important PPMC responsibilities include:
- Looking after Amoro's trademarks and branding
- Writing and submitting Incubator reports

File: amoro-site/content/community/join-community.md [hunk 4]
```diff
@@ -44,32 +43,19 @@ Everyone who contributes can become an Amoro contributor. The members will provi
...
-- Be awarded an Amoro Active Contributor e-certificate
 
 ## Committer
-Committers are promoted from Active Contributors. They have the authority to merge master branches and are responsible for the planning and maintenance of Amoro. They are also active members who share their knowledge with the community.
+Committers are promoted from Contributors. They have the authority to merge master branches and are responsible for the planning and maintenance of Amoro. They are also active members who share their knowledge with the community.
 ### How to become a Committer?
 - Have a deep understanding of Amoro's principles and future plans
 - Have the ability to deal with various issues that arise in the project promptly
 - Lead a major development, write and revise related documents
 - Be voted in by the Amoro PPMC
```
Comment (justinmclean) [inline amoro-site/content/community/join-community.md:53]: Including other forms of contribution would be a good idea, but that can be done later in another PR.

File: amoro-site/content/community/join-community.md [hunk 5]
```diff
@@ -44,32 +43,19 @@ Everyone who contributes can become an Amoro contributor. The members will provi
 - Participate in community events (meetups, hackathons, etc.)
```
Comment (justinmclean) [inline amoro-site/content/community/join-community.md:43]: and the project's mailing lists and other communication channels

File: amoro-site/content/community/join-community.md (hunk 4 above)
Comment (zhoujinsong) [reply amoro-site/content/community/join-community.md:53]: Okay, I will add more forms of contribution here later.

**Comment by zhoujinsong** [comment]
@justinmclean Thanks a lot for your reviewing!
I have made some changes according to the comments, PTAL.

**Comment by justinmclean** [comment]
Looks good to me.

**Review by zhouxinyu — APPROVED** [review]
LGTM~

**Pull request merged**